### PR TITLE
Evelyn Cordeiro - Celo Sage - Como usar a metodologia de gestão de produtos para criar soluções com NFT

### PR DESCRIPTION
**Pull request by evelyncordeiro**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [X] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [X] The title of the Pull Request matches the title of my tutorial.
- [X] I've followed the tutorial template structure.
- [X] I've proofread the tutorial for spelling and grammar mistakes.
- [X] I've thoroughly explained the code, with inline code comments and text explanations.
- [X] I've run a linter & prettifier to format my code blocks.
- [X] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/02-12-2023-Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT/Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+title: Como usar a metodologia de gestão de produtos para criar soluções com NFT
+description: Aprenda como usar a metodologia de gestão de produtos para criar soluções com NFT
+authors:
+- name: Evelyn Cordeiro
+  title: Founder of Inspiravalley
+  url: https://github.com/evelyncordeiro
+  image_url: https://avatars.githubusercontent.com/u/108406605
+tags: [beginner, celosage, erc721, smartcontract]
+hide_table_of_contents: false
```
Comment (Ishanpathak1) [inline blog/02-12-2023-Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT/Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT.md:10]: This has to be set it to true

**Review by Ishanpathak1 — COMMENTED** [review]
Let me know after you change it to true, Thank You

File: blog/02-12-2023-Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT/Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT.md (hunk 1 above)
Comment (evelyncordeiro) [reply blog/02-12-2023-Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT/Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT.md:10]: I looked at the others and are False. In the video tutorial that Viral explains how to make deploy, he speaks to leave as False. Let me know if I really need to change it.

**Comment by evelyncordeiro** [comment]
Done

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**